frontend/public/components/operator-lifecycle-manager/descriptors/spec/index.tsx

```diff
@@ -8,11 +8,15 @@ import { SpecCapability, DescriptorProps, CapabilityProps } from '../types';
 import { ResourceRequirementsModalLink } from './resource-requirements';
 import { EndpointList } from './endpoint';
 import { configureSizeModal } from './configure-size';
-import { Selector, ResourceLink } from '../../../utils';
+import { Selector, ResourceLink, LoadingInline, AsyncComponent } from '../../../utils';
+import { Tooltip } from '../../../utils/tooltip';
+import { k8sPatch } from '../../../../module/k8s';
 
-const Default: React.SFC<SpecCapabilityProps> = ({value}) => _.isEmpty(value) && !_.isNumber(value)
-  ? <span className="text-muted">None</span>
-  : <span>{value}</span>;
+const Default: React.SFC<SpecCapabilityProps> = ({value}) => {
+  return _.isEmpty(value) && !_.isNumber(value) && <span className="text-muted">None</span>
```

Comment:
I think this would be easier to read with `if` statements and separate returns for each condition.